setup.py

```diff
@@ -63,6 +63,7 @@ def get_advanced_templates():
             'ccm-deploy-test=test_util.test_aws_vpc:main',
             'test-aws-cf-deploy=test_util.test_aws_cf:main',
             'test-upgrade-vpc=test_util.test_upgrade_vpc:main',
+            'test-azure-rm-deploy=test_util.azure_test_driver:main',
```

Comment:
/me will be so happy when we have `dcos-launch destroy` as a binary instead of all these...